packages/core/src/domain/configuration/configuration.ts

```diff
@@ -52,6 +54,9 @@ export interface InitConfiguration {
   useSecureSessionCookie?: boolean | undefined
   trackSessionAcrossSubdomains?: boolean | undefined
 
+  // alternate storage option
+  allowFallbackToLocalStorage?: boolean | undefined
```

Comment:
Indeed, somehow we did not discuss it. Even though logs does not require a session storage to work, it's best to have the same one as rum if possible. This will ensure logged events will attach to the same session.
- We need to document this (`AllowFallbackToLocalStorage` should have the same value for both SDKs)
- Maybe add some E2E test to validate the behaviour